.github/actions/clang-format/entrypoint.sh

```diff
@@ -8,7 +8,17 @@ set +e
 
 # Show what would be changed as a diff.
 git-clang-format "${INPUT_TARGET}" --diffstat
-git-clang-format "${INPUT_TARGET}" --diff --quiet > auto-format-changes.diff
+
+# Fix any formatting.
+git-clang-format "${INPUT_TARGET}"
+
+# Now revert any changes to auto-generated files.
+find . -name *.pb-c.c -exec git checkout '{}' \;
+find . -name *.pb-c.h -exec git checkout '{}' \;
```

Comment:
(lint) Quote the parameter to -name so the shell won't interpret it. [SC2061]
(lint) Use ./*glob* or -- *glob* so names with dashes won't become options. [SC2035]